dGame/dUtilities/BrickDatabase.cpp

```diff
@@ -19,6 +19,11 @@ std::vector<Brick>& BrickDatabase::GetBricks(const std::string& lxfmlPath) {
 	}
 
 	AssetMemoryBuffer buffer = Game::assetManager->GetFileAsBuffer((lxfmlPath).c_str());
+
+	if (!buffer.m_Success) {
+		return emptyCache;
+	}
```

Comment:
Perhaps we should also cache an empty cache instead of just leaving early here as well?  That way we arent repeatedly checking if the file exists when it didn't exist on server start.  What are your thoughts on this?

Reply:
Agreed, but was just following convention how how it was already being handled. Out of scope to change it when this is just to prevent potential crashes